src/GameShelf.js

```diff
@@ -59,7 +59,18 @@ export default class GameShelf extends Component {
   shuffle = (games) => {
     let randoArr = games.sort(() => 0.5 - Math.random());
     return [randoArr[0]];
-  }
+	}
```

Comment:
Good idea. I think it's way better to have this split up into a different method rather than clogging up our render function